### PR TITLE
Use the common Cosmos emulator template.

### PR DESCRIPTION
**Pull request by mitchdenny**
This PR cuts over to using the shared cosmos emulator template that is now in the azure-sdk-tools repository.

### CODE REVIEW

File: eng/pipelines/templates/stages/cosmos-sdk-client.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ stages:
           vmImage: $(OSVmImage)
 
         steps:
-          - template: ../steps/cosmos-emulator.yml
+          - template: ../../../common/pipelines/templates/steps/cosmos-emulator.yml
```
Comment (weshaggard) [inline eng/pipelines/templates/stages/cosmos-sdk-client.yml:44]: Lets use `/eng/common/pipelines/templates/steps/cosmos-emulator.yml` for the path.

Comment (mitchdenny) [reply eng/pipelines/templates/stages/cosmos-sdk-client.yml]: Done. Submitted an PR for Java to tweak this as well.

Comment (mitchdenny) [reply eng/pipelines/templates/stages/cosmos-sdk-client.yml]: Done.

**Pull request merged**